src/composables/changeTheme.js

```diff
@@ -0,0 +1,11 @@
+function useThemeChange() {
+  if (theme == "light") {
+    localStorage.setItem("appTheme", "dark");
+    () => "dark";
```

Comment:
This code will bring problems during use.
`callback("dark")` use this 
same with line 7

Reply:
`callback` is an actually function, if it's getting you confused

Reply:
@annonymousauthority should the way the function been called in the  page.js change since there is no state in the function